frappe/public/js/frappe/form/controls/link.js

```diff
@@ -296,6 +297,13 @@ frappe.ui.form.ControlLink = frappe.ui.form.ControlData.extend({
 		// returns [{value: 'Manufacturer 1', 'description': 'mobile part 1, mobile part 2'}]
 	},
 
+	remove_href(doctype){
+		if(frappe.model.can_select(doctype)) {
```

Comment:
this should be set / unset based on permissions. Also what was removed in one doc, maybe be accessible in another!

Reply:
fixed with https://github.com/frappe/frappe/pull/12063/commits/87331ea107f0bf0979a7dbf536fe128e3d732111